lib/raven.interfaces.ts

```diff
@@ -14,8 +14,7 @@ export interface IRavenInterceptorOptions {
   tags?: { [key: string]: string };
   extra?: { [key: string]: any };
   fingerprint?: string[];
-  level?: Sentry.Severity;
-  context?: 'Http' | 'Ws' | 'Rpc' | "GraphQL";
```

Comment:
breaking change

Reply:
That's okay, it's a good thing to get rid of manually providing context